week-2-practise/level-1/2-validation.js

```diff
@@ -2,7 +2,7 @@
 // TIP: You should write an expression that returns a boolean value
 
 var num = 10;
-var isBigEnough; // ONLY EDIT THIS LINE
+var isBigEnough = true; // ONLY EDIT THIS LINE
```

Comment:
Hi, the expression is supposed to be num >= 10 rather than true so that the If would be correctly evaluated even when num is not 10

Reply:
Thanks @theseafaringturtle  for your helpful comments for the return statement I do write it in separate lines but  because I use Visual Studio editor when I save the file It puts the return together with the "If" statement I think this because I have installed Prettier on it.